lib/resources/StreamVideos.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (C) 2014-present Cloudflare, Inc.
+
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+'use strict';
+
+const prototypal = require('es-class');
+const auto = require('autocreate');
+
+const Resource = require('../Resource');
+const method = require('../method');
+
+/**
+ * StreamVideos represents the /accout/:id/stream API endpoint.
+ *
+ * @class StreamVideos
+ * @hideconstructor
+ * @extends Resource
+ */
+module.exports = auto(
+  prototypal({
+    extends: Resource,
+    path: 'accounts',
```

Comment:
This should probably be `accountd/:accountId/stream` and the other paths updated. (I suspect the "basic" paths will just work after that)